lib/src/svg/parser_state.dart

```diff
@@ -678,6 +678,7 @@ class SvgParserState {
   /// Drive the [XmlTextReader] to EOF and produce a [DrawableRoot].
   Future<DrawableRoot> parse() async {
     for (XmlEvent event in _readSubtree()) {
+      await Future.delayed(Duration.zero);
```

Comment:
Can we just `await null` instead?

Reply:
Apparently we can and it also works, expected that to throw an error.